benchmarking/training_scripts/resnet_cifar10/resnet_cifar10.py

```diff
@@ -189,33 +182,52 @@ def objective(config):
         ).to(device)
     milestones = [25, 40]
     optimizer = torch.optim.SGD(
-        model.parameters(), lr=lr, momentum=momentum, weight_decay=weight_decay
+        model.parameters(),
+        lr=config["lr"],
+        momentum=config["momentum"],
+        weight_decay=config["weight_decay"],
     )
     scheduler = torch.optim.lr_scheduler.MultiStepLR(
         optimizer, milestones=milestones, gamma=0.1
     )
+    return model, optimizer, scheduler
+
 
-    # Checkpointing
+def objective(config):
+    torch.manual_seed(np.random.randint(10000))
```

Comment:
Should we set a manual seed here?

Reply:
This is not my code. I think Aaron wrote it.
We control seeds of everything on our side, but not what happens in the training functions.

Reply:
We definitely should not use the same seed, because that means that initial weights and batch ordering are the same for each evaluation, that would not be good.

Reply:
In the end, we cannot assume that we can control this for some user code, so let us not try to do it.